Dockerfile

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project
+
+# This Dockerfile should be run from the parent directory of the egeria-connector-sas-viya directory
+# ie
+# docker -f ./Dockerfile 
+
+FROM odpi/egeria
+
+# Thes are optional tags used to add additional metadata into the docker image
+# These may be supplied by the pipeline in future - until then they will default
+
+ARG version=2.10-SNAPSHOT
+ARG VCS_REF=unknown
+ARG VCS_ORIGIN=unknown
+ARG BUILD_TIME=unknown
+ARG VCS_DATE=unknown
+
+ENV version ${version}
+
+
+# Labels from https://github.com/opencontainers/image-spec/blob/master/annotations.md#pre-defined-annotation-keys (with additions prefixed    ext)
+LABEL org.opencontainers.image.vendor = "ODPi" \
+      org.opencontainers.image.title = "Egeria" \
+      org.opencontainers.image.description = "Egeria with SAS Viya connector" \
+      org.opencontainers.image.url = "https://egeria.odpi.org/" \
+      org.opencontainers.image.source = "$VCS_ORIGIN" \
+      org.opencontainers.image.authors = "ODPi Egeria" \
+      org.opencontainers.image.revision = "$VCS_REF" \
+      org.opencontainers.image.licenses = "Apache-2.0" \
+      org.opencontainers.image.created = "$BUILD_TIME" \
+      org.opencontainers.image.version = "$version" \
+      org.opencontainers.image.documentation = "https://github.com/odpi/egeria-connector-sas-viya" \
```

Comment:
wrong url - shouldn't it point to this repo?

Reply:
That url is to this repo isn't it?